ang/test/civicase/case/search/directives/search.directive.spec.js

```diff
@@ -156,8 +156,8 @@
             $scope.$digest();
           });
 
-          it('does not calls $scope.doSearch()', () => {
-            expect($scope.doSearch).not.toHaveBeenCalled();
+          it('calls $scope.doSearch()', () => {
```

Comment:
These unit tests descriptions are too technical. Can we remove statements like `calls $scope.doSearch()` and replace with `searches for new cases` or something usual.
I know this was not done in this PR, but would be great if we can fix it for the whole file.

Reply:
@deb1990 did a few fixes here:

- We didn't have proper tests for most scope variables. Added them.
- Removed technical references from the test files.
- The `caseManagerIsMe` function was returned either `true`, `false`, or `undefined` depending on a filter. We now return `false` instead of `undefined`.
- Removed the `mapSelectOptions()` test because this function is not even exposed, and the test was not properly implemented.